mobile/lib/shared/services/asset.service.dart

```diff
@@ -66,8 +66,11 @@ class AssetService {
     try {
       final etag = hasCache ? _db.eTags.getByIdSync(user.id)?.value : null;
       final (List<AssetResponseDto>? assets, String? newETag) =
-          await _apiService.assetApi
-              .getAllAssetsWithETag(eTag: etag, userId: user.id);
+          await _apiService.assetApi.getAllAssetsWithETag(
```

Comment:
strange formatting?

Reply:
all good, it's dart's auto-formatter and you find it everywhere in the dart codebase :-)